src/Guide/Main.hs

```diff
@@ -73,7 +72,7 @@ import qualified Data.Text as T
 import qualified Data.Text.IO as T
 import qualified Network.Wai.Metrics as EKG
 import qualified SlaveThread as Slave
-import qualified System.FSNotify as FSNotify
+--import qualified System.FSNotify as FSNotify
```

Comment:
You can just remove this line.

Reply:
And probably also remove `fsnotify` (or whatever it's called) from our cabal and stack files.